AmethystLauncher/src/loader/Loader.cpp

```diff
@@ -95,9 +91,18 @@ void ModLoader::getMinecraftWindowHandle()
     mMinecraftWindowHandle = procHandle;
 }
 
-void ModLoader::InjectRuntime()
-{
-	// Get the path to the amethyst runtime dll
+void ModLoader::InjectRuntime() {
+    std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+
+    if (mConfig.injectedMod.length() == 0) {
+        std::string message = fmt::format(
+            "No injected mod specified. Make sure to update your config.json "
+            "with the latest Amethyst Runtime.");
```

Comment:
Does having two strings side by side work?

Reply:
Yes, although let me remove that.

Reply:
Okay removed

Reply:
Huh lol, weird